source-map.bs

```diff
@@ -352,20 +352,20 @@ Explorer's Conditional Compilation and was changed to `//#`.  It is reasonable f
 also accept `//@` but `//#` is preferred.
 
 This recommendation works well for JavaScript, it is expected that other source files will
-have other conventions.  For instance for CSS `/*# sourceMappingURL=<url> */` is proposed.
+have different conventions.  For instance, for CSS `/*# sourceMappingURL=<url> */` is proposed.
 
 `<url>` is a URL as defined in [[URL]]; in particular,
 characters outside the set permitted to appear in URIs must be percent-encoded
-and it maybe a data URI.  Using a data URI along with [=sourcesContent=] allow
-for a completely self-contained source-map.
+and it may be a data URI.  Using a data URI along with [=sourcesContent=] allows
+for a completely self-contained source map.
 
 Regardless of the method used to retrieve the [=Source Mapping URL=] the same
 process is used to resolve it, which is as follows:
 
 When the [=Source Mapping URL=] is not absolute, then it is relative to the generated code's
 <dfn>source origin</dfn>. The [=source origin=] is determined by one of the following cases:
 
-- If the generated source is not associated with a script element that has a `src`
+- If the generated source is not associated with a script element that has an `src`
```

Comment:
I assume this depends on if we read it as "S-R-C" or as "source". I'd personally always read it as "source" because it's not initials and not usually uppercase these days. But no strong preference.

Reply:
Agree, it's also ready it as "source"